controller/auth.js

```diff
@@ -0,0 +1,35 @@
+const db = require("../models/index.js");
+const models = db.sequelize.models;
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcryptjs');
+require('dotenv').config({ slient: true });
+
+async function login (req, res) {
+  if (req.user)
+    return res.status(400).send("Already signed in");

```

Comment:
E pull master về sẽ thấy chị define status code constant. Em cùng status code luôn nhé. Thank you em :D